www/source/tutorials/getting-started/windows/setup-environment.html.md.erb

```diff
@@ -0,0 +1,41 @@
+---
+title: Getting set up with Habitat | Mac
+---
+
+<ul class="main-content--version">
+  <li>Current version:</li>
+  <li><a href="/tutorials/getting-started/mac/setup-environment">Mac</a></li>
+  <li><a href="/tutorials/getting-started/linux/setup-environment">Linux</a></li>
+  <li><a class="active" href="/tutorials/getting-started/windows/setup-environment">Windows</a></li>
+</ul>
+<hr />
+
+# Set up your environment
+The `hab` command-line interface (CLI) tool downloads its other components when it needs them, so setup for Habitat is centered on where you want the `hab` CLI installed and getting it integrated into your shell.
+
+1. Open a PowerShell window. Change directory to the directory where you want to place the `hab` CLI and copy the `hab` CLI to that directory.
+
+        mkdir ~/bin
+        cp ./where/you/extracted/hab -Destination ~/bin
+
+2. Add the location for the `hab` CLI to your `PATH` environment variable.
+
+       $env:PATH = $env:PATH;$env:USERPROFILE/bin
```

Comment:
this should be `$env:PATH = "$env:PATH;$env:USERPROFILE/bin"` (just missing the double quotes around the value to the right of the equals sign.